txn_writer.go

```diff
@@ -21,15 +21,63 @@ import (
 // Example string to be written to the manifest file
 // if no dependencies are found in the project
 // during `dep init`
-const exampleToml = `
-# Example:
+const exampleTOML = `
+## EXAMPLE (these lines may be deleted)
+#
+## "required" lists a set of packages (not projects) that must be included in
```

Comment:
I can see how that makes sense with the `[[dependencies]]`, where it looks more or less like a section header. But with I think it'd be more confusing to do that on individual values. I can't think of a commented config file I've seen that puts descriptive docs below the values they describe.